src/components/custom/_signup-affiliate.js

```diff
@@ -0,0 +1,217 @@
+    }
+    @media ${device.mobileL} {
+        width: 100%;
+        padding: 6rem 2rem;
+    }
+`
+const StyledHeader = styled(Header)`
+    @media ${device.tabletL} {
+        font-size: 3rem;
+        margin-bottom: 3rem;
+    }
+`
+
+const SubTitle = styled(Text)`
+    @media ${device.tabletL} {
+        font-size: 2rem;
+        margin-bottom: 1rem;
+    }
+`
+const StyledText = styled(Text)`
+    @media ${(props) => device.tabletL && props.notedBox} {
+        width: 13rem;
+    }
+    @media (max-width: 340px) {
+        width: 17rem;
+    }
+    @media ${device.tabletL} {
+        font-size: ${(props) => props.tabletFontSize || 'var(--text-size-xxs)'};
+    }
+`
+const NoteBox = styled.div`
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    min-height: 1.7rem;
+    padding: 8px 14px;
+    margin-top: 1.6rem;
+    border-radius: 0.4rem;
+    background-color: rgba(242, 243, 244, 0.56);
+
+    @media ${device.mobileL} {
+        min-height: auto;
+        padding: 13px 16px;
+    }
+`
+const InputGroup = styled.div`
+    position: relative;
+    width: 100%;
+    margin: 2.5rem 0 0.6rem;
+
+    @media ${device.mobileL} {
+        margin: 25px 0 16px 0;
+    }
+`
+const EmailButton = styled(Button)`
+    width: 100%;
+    font-size: 1.4rem;
+    margin-bottom: 0.4rem;
+    margin-top: 3.2rem;
+
+    @media ${device.tabletL} {
+        margin-top: 24px;
+    }
+
+    @media ${device.mobileL} {
+        font-size: 1.75rem;
+    }
+`
+const LoginText = styled(Text)`
+    text-align: center;
+    align-self: center;
+    margin-top: 1.6rem;
+    font-size: 15px;
+
+    @media ${device.tabletL} {
+        margin-bottom: 0;
+        margin-top: 3.75rem;
+    }
+    @media ${device.tabletL} {
+        font-size: 2rem;
+    }
+`
+const StyledLinkText = styled(LinkText)`
+    font-size: ${(props) => props.size || '14px'};
+`
+const SignupAffiliate = ({
+    autofocus,
+    clearEmail,
+    email,
+    email_error_msg,
+    handleInputChange,
+    handleLogin,
+    handleValidation,
+    is_ppc,
+    is_submitting,
+}) => {
+    const [is_checked, setChecked] = useState(false)
+
+    const handleChange = (event) => {
+        setChecked(event.currentTarget.checked)
+    }
+
+    return (
+        <SignupContent>
+            <StyledHeader as="h4" type="sub-section-title" mb="0.8rem">
+                {localize('Sign up')}
+            </StyledHeader>
+            <SubTitle>{localize('Enter your email address to begin')}</SubTitle>
+
+            {!is_ppc && (
+                <NoteBox>
+                    <div>
+                        <StyledText
+                            mb="0.4rem"
+                            notedBox
+                            color="grey-16"
+                            size="var(--text-size-xs)"
+                            tabletFontSize="12px"
+                        >
+                            {localize('Want to sign up as a trader?')}
+                            <StyledLinkText
```

Comment:
If this is used to navigate users to another link, you should be using `LocalizedLink` instead of `LinkText` here.